public/docs/_examples/structural-directives/dart/pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: structural_directives
+description: Structural directives example
```

Comment:
There was a mention on another PR and the comments sounded that SDK constraint `>=1.13.0<2.0.0` makes sense to ensure `^1.2.3` version constraints in `pubspec.yaml` are supported as well as `async` / `await`. 
See https://github.com/angular/angular.io/pull/693#discussion_r49625450 and follow-ups.
No definitive yes/or no though.